CoreScriptsRoot/Modules/Settings/Pages/LeaveGame.lua

```diff
@@ -23,13 +24,20 @@ local PageInstance = nil
 RobloxGui:WaitForChild("Modules"):WaitForChild("TenFootInterface")
 local isTenFootInterface = require(RobloxGui.Modules.TenFootInterface):IsEnabled()
 
+local useShutdownToLeaveSuccess,useShutdownToLeaveValue = pcall(function() settings():GetFFlag("ChatLayoutChange") return end) 
+local useShutdownToLeave = useShutdownToLeaveSuccess and useShutdownToLeaveValue
 
 ----------- CLASS DECLARATION --------------
 
 local function Initialize()
 	local settingsPageFactory = require(RobloxGui.Modules.Settings.SettingsPageFactory)
 	local this = settingsPageFactory:CreateNewPage()
 
+	this.LeaveFunc = function()
+		GuiService.SelectedCoreObject = nil -- deselects the button and prevents spamming the popup to save in studio when using gamepad
```

Comment:
It's not serialized, afak, but I did this so that it would deselect the button the make it appear more responsive.